content/rc/databases/active-active-redis.md

```diff
@@ -12,28 +12,28 @@ aliases: [
 
-## Active-Active geo-distributed replication
-
-Active-Active databases use both clustering and replication to strengthen your database. Active-Active Redis has additional features like geo-distribution, multiple active proxies, conflict resolution, and automatic failover to provide you with a more durable and scalable database.
+## Active-Active geo-distributed replication highlights
 
 ### Multi-zone
 
 Geo-distributed replication maintains copies of both primary and replica shards in multiple clusters. These clusters can be spread across multiple availability zones. Active-Active Redis uses zone awareness to spread your primary and replica shards across zones, which helps protect against data loss from regional outages.
 
-### Multiple active proxies
+### Local latency with unique endpoints
 
-Active-Active databases use a multi-primary architecture, which lets you read and write to a primary shard in any of your participating clusters. Having [multiple active proxies]({{<relref "/rs/databases/configure/proxy-policy#about-multiple-active-proxy-support">}}) allows users to connect to the cluster closest to them, reducing latency.
+Applications can connect to a specific copy of an Active-Active database using its unique endpoint. For local latency, configure your application to use a database endpoint in the closest region.
 
 ### Conflict resolution
 
 Active-Active databases use special data types called conflict-free data types (CRDT). These automatically resolve conflicts that occur when writes are made to different clusters at the same time.
 
-### Automatic failover
+### Failover handling
+
+After a failure at the process, node, or zone level, Active-Active databases automatically promote replica shards to replace failed primaries, copy data to new replica shards, and migrate shards to new nodes as needed. This reduces downtime and makes the most of your computing resources, even in the event of a failure.  
 
-After a failure at any level (process, node, or zone), Active-Active databases automatically promote replica shards to replace failed primaries, copy data to new replica shards, and migrate shards to new nodes as needed. This reduces downtime and makes the most of your computing resources, even in the event of a failure.  
+Active-Active also provides a failover opportunity in the event an entire cluster fails.  Applications that are connected to a cluster in a state of failure can be programmed to redirect traffic to a non-failed state Active-Active connected cluster.  When the original cluster recovers, the application can direct traffic back to the original cluster.  Redirecting the application traffic to alternate clusters is not handled automatically by Active-Active.  However, data will automatically sync to the recovered clusters when it returns to a healthy state.
```

Comment:
```suggestion
Active-Active also provides a failover opportunity in case an entire cluster fails. Applications should redirect traffic to another cluster in the Active-Active deployment if a connected cluster fails. After the original cluster recovers, applications can direct traffic back to the original cluster. Redirecting traffic to alternate clusters is not handled automatically by Active-Active, so you must add redirection logic to the application. However, data automatically syncs to a recovered cluster when it returns to a healthy state.
```